src/app/api/activity/activity.listener.ts

```diff
@@ -0,0 +1,17 @@
+import { EventEmitter } from 'node:events'
+import User from '../core/models/User.model'
+import { UpdateTaskRequest } from '@/types/dto/tasks.dto'
+import { Task } from '@prisma/client'
+import { ActivityLogger } from './activityLogger.service'
+
+export const activityEvents = new EventEmitter()
+
+activityEvents.on('post', async (taskId: string, user: User) => {
```

Comment:
later there can be multiple post endpoints, so be more precise, we can name it as `ACTIVITY_LOG_CREATED`

Reply:
updated in 3257f28c5c3a5be4bac62db9f70b1d7deef4908d commit